crates/astria-sequencer-relayer/src/validator.rs

```diff
@@ -1,36 +1,34 @@
 use std::path::Path;
 
+use astria_core::crypto::SigningKey;
 use astria_eyre::eyre::{
     self,
     bail,
     WrapErr as _,
 };
-use ed25519_consensus::{
-    SigningKey,
-    VerificationKey,
-};
+use ed25519_consensus::VerificationKey;
 use tendermint::account;
 use tendermint_config::PrivValidatorKey;
 use tracing::instrument;
-use zeroize::{
-    Zeroize,
-    ZeroizeOnDrop,
-};
 
 /// `Validator` holds the ed25519 keys to sign and verify tendermint
 /// messages. It also contains its address (`AccountId`) in the tendermint network.
-#[derive(Clone, Debug, Zeroize, ZeroizeOnDrop)]
+#[derive(Clone, Debug)]
 pub(crate) struct Validator {
     /// The tendermint validator account address; defined as
     /// Sha256(verification_key)[..20].
-    #[zeroize(skip)]
     pub(crate) address: account::Id,
 
     /// The ed25519 signing key of this validator.
+    // allow: this entire struct is due to get removed as part of
+    // https://github.com/astriaorg/astria/issues/1010
+    #[allow(dead_code)]
```

Comment:
IMO go ahead and remove this. The fact that `Validator` is pub-crate to being with is an oversight (or maybe due to how the integration tests are written).

Nobody should use `astria-sequencer-relayer` as a direct dependency.

Reply:
Done in [6b05e11](https://github.com/astriaorg/astria/pull/1074/commits/6b05e114d3a3860868252108f2fc7be22636e433).